nengo/tests/conftest.py

```diff
@@ -232,6 +233,11 @@ def pytest_runtest_setup(item):
         if skip:
             pytest.skip(" and ".join(skipreasons))
 
+    if 'Simulator' in item.fixturenames:
+        for test, reason in _Simulator.unsupported:
+            if re.match(test, item.name):
```

Comment:
We could also write our own function to make a unique name for test functions. I do something like that above in `function_seed`, and it would be great to use the same code both here and there if possible. For this feature, it's especially important that the unique name is easily readable and writeable. I actually quite like `item.nodeid` for function names, though you're right it gets weird for classes (though I'm not sure how much we have to worry about classes, since I don't think we ever use them).